### ISSUE TITLE
OS never tries to self-heal after losing network [VM]

### ISSUE THREAD

**Issue by statusquont** [mynodebtc/mynode#839]
**Describe the bug**
I run the latest VM version. There are times when myNode will somehow lose it's local IP from the host. I haven't been able to pin point why, for as far as I can tell, the host machine still has internet access. Sometimes the VM will have a local IP for days at a time, sometimes it loses it within a hour. The only solution is to completely shut down the VM, and restart it which is a bit annoying and time consuming.

**Expected behavior**
It would be nice if there was a one-click "fix it" button next to the Local IP field on the settings page IF it doesn't have an IP. It's nice that it at least acknowledges there is an error, now help the user fix it -- ideally without a full reboot.

**Screenshots**
![error](https://github.com/mynodebtc/mynode/assets/15948511/8f851716-101f-4568-b326-5982af288a08)

**Desktop (please complete the following information):**
 - Browser: n/a
 - v0.3.24
 - VirtualBox 7.0

**MyNode hardware (please complete the following information):**
 - Device:  ThinkPad X1 Carbon Gen 7

**Comment by tehelsper** [comment]
You may be able to force it to renew with "sudo dhclient". Alternatively, you could dig more into the NetworkManager settings or log a support ticket.

**Comment by statusquont** [comment]
I just had a chance to give this a try, and when I open the Linux Terminal from the /status page it prompts for the login credentials, which I enter correctly and then it takes me to the URL:  localhost:2222/ssh/host/error and so there is no way to enter that command.

But then, I minimized the browser in the VM window, and right-clicked the desktop to bring up Terminal Emulator, from there I entered the `sudo dhclient` command, and it worked! It did pick up a different IP address than what it had before, but at least this will be a _huge_ time saver in the future and allow me to bring it back online without a full reboot! 

Grateful!
